src/app/beer_garden/db/mongo/util.py

```diff
@@ -133,7 +102,7 @@ def ensure_users(guest_login_enabled):
         Principal(
             username="admin",
             hash=custom_app_context.hash(default_password),
-            roles=[LegacyRole.objects.get(name="bg-admin")],
```

Comment:
This reference to one of the old LegacyRole's, which I removed as part of this branch, was an issue that I didn't catch until I tested starting from a clean DB.  This entire `ensure_users` function will go or be rewritten at some point, but for now I've opted to make the minimum changes necessary to have the code not blow up.